docs/kubernetes/flannel-bigip-info.rst

```diff
@@ -0,0 +1,116 @@
+.. index::
+   single: BIG-IP; flannel; VXLAN; BIG-IP Controller; Kubernetes; Cluster Network
+
+.. _flannel-bigip-info:
+
+BIG-IP device integration with flannel VXLAN in Kubernetes
+==========================================================
+
+Overview of Cluster Networking with flannel in Kubernetes
+---------------------------------------------------------
+
+.. sidebar:: :fonticon:`fa fa-info-circle` Related
+
+   Read about the Kubernetes `Cluster Network`_ and `Using flannel with Kubernetes`_.
+
+`Flannel`_ is a layer 3 network fabric (or, in their words, "a virtual network that attaches IP addresses to containers"). Flannel assigns a subnet to each Node and allocates an IP address within the Node's subnet to each of the Node's Pods.
+
+For Kubernetes v1.6 and later, flannel provides a ``kubeadm`` network add-on. When you apply the `flannel manifest`_ in Kubernetes, the flannel Pod deploys on each Node in the Cluster. The Pod consists of two containers:
```

Comment:
Can we remove the kubeadm reference? The flannel manifest file can be used without kubeadm. There is no need to reference it here. Everything else in the paragraph is fine.